client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -897,10 +907,93 @@ class CancelPurchaseForm extends Component {
 		return parseFloat( refundAmount ).toFixed( precision );
 	};
 
+	renderBenefitsGrid = () => {
+		const { purchase, translate } = this.props;
+
+		switch ( purchase.productSlug ) {
+			case 'business-bundle':
+				return (
+					<div className={ 'cancel-purchase-form__benefits-wrapper' }>
+						<div className={ 'cancel-purchase-form__benefits-wrapper-grid' }>
+							{ /* TODO: translate the below */ }
+							<div>
+								{ /* would be good not to hardcode this */ }
+								<h3>200GB of storage</h3>
+								{ /*  Not sure about this - is using USD ok when customers are using other currencies? */ }
+								<p>
+									Other hosting providers charge $25 or more per month for storage increases over
+									100GB
+								</p>
+								<p>After removing plan:</p>
+								{ /* would be good not to hardcode this */ }
+								<p>3 GB File Storage</p>
+							</div>
+							<div>
+								<h3>Design Customization</h3>
```

Comment:
This feels vague and not strictly true - you do have some customisation options - you can change theme, what you don't have (mainly) is the ability to customise css. 

On the plans page for new sites, we no longer make a song and dance about this difference.